msteams-platform/tabs/what-are-tabs.md

```diff
@@ -1,77 +1,72 @@
 ---
-title: What are custom tabs in Teams?
+title: Microsoft Teams tabs
 author: surbhigupta
 description: An overview of custom tabs on the Teams platform
 localization_priority: Normal
 ms.topic: overview
 ms.author: lajanuar
 ---
+
 # Microsoft Teams tabs
 
 Tabs are Teams-aware webpages embedded in Microsoft Teams. They are simple HTML <iframe\> tags that point to domains declared in the app manifest and can be added as part of a channel inside a team, group chat, or personal app for an individual user. You can include custom tabs with your app to embed your own web content in Teams or add Teams-specific functionality to your web content. For more information, see [Teams JavaScript client SDK](/javascript/api/overview/msteams-client).
 
-There are two types of tabs available in Teams — channel/group and personal. Channel/group tabs deliver content to channels and group chats, and are a great way to create collaborative spaces around dedicated web-based content. Personal tabs, along with personally-scoped bots, are part of personal apps and are scoped to a single user. They can be pinned to the left navigation bar for easy access.
+The following image shows Contoso channel tabs:
+
+![Channel or group and personal tabs](../assets/images/tabs/tabs.png)
+
+There are few prerequisites that you must go through before working on tabs.
+
```

Comment:
Can we move this sentence after the video? currently it feels like videos are about prerequesites.